umake/frameworks/__init__.py

```diff
@@ -153,6 +154,8 @@ def __init__(self, name, description, category, force_loading=False, logo_path=N
         self.packages_requirements.extend(self.category.packages_requirements)
         self.only_for_removal = only_for_removal
         self.expect_license = expect_license
+        self.version_regex = version_regex
+        self.forbidden_to_update = False
```

Comment:
This was my way of preventing an update on some frameworks like Intellij.
Set by default at False then overridden in the class BaseJetBrains for example.

Reply:
I would have used it the other way around, keeping the default as not updatable, and setting a framework to updatable explicitly. This way we only add information for those that have this functionality.

What do you think?